Development environment/Laravel Homestead setup.md

```diff
@@ -0,0 +1,147 @@
+Homestead is an official, pre-packaged Vagrant box from Laravel team. Its default approach is having multiple sites set up on a single Vagrant instance, which makes adding new sites a trivial task. For more advanced requirements there is an option to set up each site on its own instance.
+
+Homestead runs on any Windows, Mac, or Linux system, and includes the Nginx web server, PHP 7.3, PHP 7.2, PHP 7.1, MySQL, PostgreSQL, Redis, Memcached, Node, etc.. For full list of features check out the [offical documentation](https://laravel.com/docs/5.8/homestead)
+
+## Installing Laravel Homestead
+
+As any other Vagrant boxes, Homestead requires a few prerequisites installed:
+
+* [Vagrant](https://www.vagrantup.com/downloads.html)
+* [Git](https://git-scm.com/downloads)
+* Either one of the following: [VirtualBox](https://www.virtualbox.org/wiki/Downloads), [VMWare](https://my.vmware.com/en/web/vmware/downloads)<sup>1</sup>, [Parallels](https://www.parallels.com/products/desktop/)<sup>2</sup>, or [Hyper-V](https://docs.microsoft.com/en-us/virtualization/hyper-v-on-windows/quick-start/enable-hyper-v)
+
+<sup>1 Requires the purchase of both VMware Fusion / Workstation and the VMware Vagrant plug-in.</sup>
+
+<sup>2 Not free, but has a free trial period and requires a free Vagrant plug-in</sup>
```

Comment:
you can add here: "if you want to use it this plugin can be purchased from your educational budget. For more informations talk to your team lead."

Reply:
:+1